.editorconfig

```diff
@@ -0,0 +1,7 @@
+[*]
+indent_style = space
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
wrong add?

Reply:
I wanted the editor to clear the spaces at the end of the lines (but not on markdown) so I added it